### PR TITLE
fix job and context name for jasmine tests

### PR DESCRIPTION
**Pull request by rycornell**
Remove `Jasmine__TopLevel__Suite` from the context and use test.title since test.name is undefined

### CODE REVIEW

File: lib/sauce-service.js [hunk 1]
```diff
@@ -34,11 +36,14 @@ class SauceService {
          * framework hooks in order to execute async functions.
          * This tweak allows us to set the real suite name for jasmine jobs.
          */
-        if (this.suiteTitle === 'Jasmine__TopLevel__Suite') {
-            this.suiteTitle = test.fullName.slice(0, test.fullName.indexOf(test.name) - 1)
+
```
Comment (christian-bromann) [inline lib/sauce-service.js]: unnecessary new line

File: lib/sauce-service.js [hunk 2]
```diff
@@ -34,11 +36,14 @@ class SauceService {
...
          * This tweak allows us to set the real suite name for jasmine jobs.
          */
-        if (this.suiteTitle === 'Jasmine__TopLevel__Suite') {
-            this.suiteTitle = test.fullName.slice(0, test.fullName.indexOf(test.name) - 1)
+        
+        if (this.suiteTitle === jasmineTopLevelSuite) {
+            this.suiteTitle = test.fullName.slice(0, test.fullName.indexOf(test.title) - 1)
         }
+        
+        var context = test.parent === jasmineTopLevelSuite ? test.fullName : test.parent + ' - ' + test.title
```
Comment (christian-bromann) [inline lib/sauce-service.js]: `const`

**Review by christian-bromann — CHANGES_REQUESTED** [review]
Two comments .. also since this package is already ported, please also do a PR in the new [v5](https://github.com/webdriverio/v5/tree/master/packages/wdio-sauce-service) repo. Thanks!

File: lib/sauce-service.js (hunk 2 above)
Comment (rycornell) [reply lib/sauce-service.js]: Thanks, I didn't know `const` was valid here.

**Review by christian-bromann — APPROVED** [review]
👍 

Will merge as soon as it is ported to v5

**Comment by christian-bromann** [comment]
Thanks for applying the patch twice!

**Comment by christian-bromann** [comment]
Released as `v0.4.9`

**Pull request merged**